{"level":"error","message":"Error creating the user: TypeError: db.selecte is not a function","service":"DevOps-api"}
{"level":"error","message":"signup error db.selecte is not a function","service":"DevOps-api","stack":"TypeError: db.selecte is not a function\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:18:8)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:24)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:295:15\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)"}
{"level":"error","message":"Error creating the user: TypeError: db.selecte is not a function","service":"DevOps-api"}
{"level":"error","message":"signup error db.selecte is not a function","service":"DevOps-api","stack":"TypeError: db.selecte is not a function\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:18:8)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:24)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:295:15\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user db.selecte is not a function","service":"DevOps-api","stack":"TypeError: db.selecte is not a function\n    at selecte (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:55:8)\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:24)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:295:15\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at Function.handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:186:3)\n    at router (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:60:12)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at logger (E:\\Desktop\\DevOps-1\\node_modules\\morgan\\index.js:144:5)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at cookieParser (E:\\Desktop\\DevOps-1\\node_modules\\cookie-parser\\index.js:57:14)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at urlencodedParser (E:\\Desktop\\DevOps-1\\node_modules\\body-parser\\lib\\types\\urlencoded.js:57:7)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at E:\\Desktop\\DevOps-1\\node_modules\\body-parser\\lib\\read.js:132:5\n    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)\n    at invokeCallback (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:238:16)\n    at done (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:519:28)\n    at endReadableNT (node:internal/streams/readable:1696:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user db.selecte is not a function","service":"DevOps-api","stack":"TypeError: db.selecte is not a function\n    at selecte (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:55:8)\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:24)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:157:13)\n    at Route.dispatch (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\route.js:117:3)\n    at handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:435:11)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:295:15\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at Function.handle (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:186:3)\n    at router (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:60:12)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at logger (E:\\Desktop\\DevOps-1\\node_modules\\morgan\\index.js:144:5)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at cookieParser (E:\\Desktop\\DevOps-1\\node_modules\\cookie-parser\\index.js:57:14)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at urlencodedParser (E:\\Desktop\\DevOps-1\\node_modules\\body-parser\\lib\\types\\urlencoded.js:57:7)\n    at Layer.handleRequest (E:\\Desktop\\DevOps-1\\node_modules\\router\\lib\\layer.js:152:17)\n    at trimPrefix (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:342:13)\n    at E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:297:9\n    at processParams (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:582:12)\n    at next (E:\\Desktop\\DevOps-1\\node_modules\\router\\index.js:291:5)\n    at E:\\Desktop\\DevOps-1\\node_modules\\body-parser\\lib\\read.js:132:5\n    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)\n    at invokeCallback (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:238:16)\n    at done (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:227:7)\n    at IncomingMessage.onEnd (E:\\Desktop\\DevOps-1\\node_modules\\raw-body\\index.js:287:7)\n    at IncomingMessage.emit (node:events:519:28)\n    at endReadableNT (node:internal/streams/readable:1696:12)\n    at processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test@example.com,1","service":"DevOps-api"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"signup error Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test@example.com,1","params":["test@example.com",1],"query":"select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"DevOps-api"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error authenticated user Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test@example.com,1","params":["test@example.com",1],"query":"select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"DevOps-api"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test1@example.com,1","service":"DevOps-api"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"signup error Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test1@example.com,1","params":["test1@example.com",1],"query":"select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"DevOps-api"}
{"cause":{"name":"NeonDbError","sourceError":{}},"level":"error","message":"Error authenticated user Failed query: select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2\nparams: test1@example.com,1","params":["test1@example.com",1],"query":"select \"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\" from \"users\" where \"users\".\"email\" = $1 limit $2","service":"DevOps-api"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:49:28)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at Object.module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:7:18)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (E:\\Desktop\\DevOps-1\\src\\utils\\jwt.js:10:13)\n    at sign (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:20:28)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm,dhjhjdfdfudjashd@gmail.com,user","service":"DevOps-api"}
{"cause":{"code":"22001","file":"varchar.c","line":"637","name":"NeonDbError","routine":"varchar","severity":"ERROR"},"level":"error","message":"signup error Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm,dhjhjdfdfudjashd@gmail.com,user","params":["dave","$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm","dhjhjdfdfudjashd@gmail.com","user"],"query":"insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"","service":"DevOps-api","stack":"Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm,dhjhjdfdfudjashd@gmail.com,user\n    at NeonHttpPreparedQuery.queryWithCache (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:26:23)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"cause":{"code":"22001","file":"varchar.c","line":"637","name":"NeonDbError","routine":"varchar","severity":"ERROR"},"level":"error","message":"Unhandled error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm,dhjhjdfdfudjashd@gmail.com,user","params":["dave","$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm","dhjhjdfdfudjashd@gmail.com","user"],"query":"insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"","service":"DevOps-api","stack":"Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$qjp.PU89D0X2rx5txibcQuu5rGhcEjXNskR3U2cSQR2waGfXdadwm,dhjhjdfdfudjashd@gmail.com,user\n    at NeonHttpPreparedQuery.queryWithCache (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:26:23)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm,dhjhjdfdfudjashd@gmail.com,user","service":"DevOps-api"}
{"cause":{"code":"22001","file":"varchar.c","line":"637","name":"NeonDbError","routine":"varchar","severity":"ERROR"},"level":"error","message":"signup error Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm,dhjhjdfdfudjashd@gmail.com,user","params":["dave","$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm","dhjhjdfdfudjashd@gmail.com","user"],"query":"insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"","service":"DevOps-api","stack":"Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm,dhjhjdfdfudjashd@gmail.com,user\n    at NeonHttpPreparedQuery.queryWithCache (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:26:23)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"cause":{"code":"22001","file":"varchar.c","line":"637","name":"NeonDbError","routine":"varchar","severity":"ERROR"},"level":"error","message":"Unhandled error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm,dhjhjdfdfudjashd@gmail.com,user","params":["dave","$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm","dhjhjdfdfudjashd@gmail.com","user"],"query":"insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"","service":"DevOps-api","stack":"Error: Failed query: insert into \"users\" (\"id\", \"name\", \"password\", \"email\", \"role\", \"createdAt\", \"updatedAt\") values (default, $1, $2, $3, $4, default, default) returning \"id\", \"name\", \"email\", \"role\", \"createdAt\"\nparams: dave,$2b$10$tZyWWeIQxcvR8cX6aD9/Y.d7m8EYroLYOJFmj2I1dEDViJO//FiLm,dhjhjdfdfudjashd@gmail.com,user\n    at NeonHttpPreparedQuery.queryWithCache (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/pg-core/session.js:41:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async NeonHttpPreparedQuery.execute (file:///E:/Desktop/DevOps-1/node_modules/drizzle-orm/neon-http/session.js:45:20)\n    at async createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:26:23)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:23:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:19:18)"}
{"level":"error","message":"Failed to create token  secretOrPrivateKey must have a value","service":"DevOps-api","stack":"Error: secretOrPrivateKey must have a value\n    at module.exports [as sign] (E:\\Desktop\\DevOps-1\\node_modules\\jsonwebtoken\\sign.js:111:20)\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:7:18)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: Failed to create token","service":"DevOps-api","stack":"Error: Failed to create token\n    at Object.sign (file:///E:/Desktop/DevOps-1/src/utils/jwt.js:10:13)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:20:28)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"signup error res.cookies is not a function","service":"DevOps-api","stack":"TypeError: res.cookies is not a function\n    at Object.set (file:///E:/Desktop/DevOps-1/src/utils/cookies.js:9:9)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:25:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Unhandled error: res.cookies is not a function","service":"DevOps-api","stack":"TypeError: res.cookies is not a function\n    at Object.set (file:///E:/Desktop/DevOps-1/src/utils/cookies.js:9:9)\n    at signup (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:25:13)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:59:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:59:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:59:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:60:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:60:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:60:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:60:30)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signIn (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:60:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:60:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Error authenticated user User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:60:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Signin error","service":"DevOps-api"}
{"level":"error","message":"Unhandled error: User not founde","service":"DevOps-api","stack":"Error: User not founde\n    at authenticateUser (file:///E:/Desktop/DevOps-1/src/services/auth.service.js:60:30)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async signIn (file:///E:/Desktop/DevOps-1/src/controllers/auth.controller.js:48:18)"}
{"level":"error","message":"Error creating the user: Error: User with this email already exists","service":"DevOps-api"}
{"level":"error","message":"signup error User with this email already exists","service":"DevOps-api","stack":"Error: User with this email already exists\n    at createUser (E:\\Desktop\\DevOps-1\\src\\services\\auth.service.js:23:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at signup (E:\\Desktop\\DevOps-1\\src\\controllers\\auth.controller.js:19:18)"}
